Curve-finance-suite/Pool-Migration/README.md

```diff
@@ -0,0 +1,29 @@
+# Pool Migration agent
+
+## Description
+
+This agent detects successful `migrate_to_new_pool` calls to Pool Migrator Contract.
```

Comment:
I am not sure if `successful` is the correct word here because I think failed transactions could also trigger your findings.

Reply:
You're correct. Just tried with tx [0x88e002177f0b0bc4becd0fc21c7445eb915c9879915ef7cf61fee15f43c9c809](https://etherscan.io/tx/0x88e002177f0b0bc4becd0fc21c7445eb915c9879915ef7cf61fee15f43c9c809) and it triggered it. Will update.